.gitattributes

```diff
@@ -0,0 +1,8 @@
+/script export-ignore
```

Comment:
Yep, scripts is for used little utilities while developing the system rather than anything depended upon by using the library.